Nichan/client

Niichan1403 opened the pull request.

Niichan1403 committed:
Create Task button

Niichan1403 committed:
Eisenhower block

Niichan1403 committed:
Fix inconsistency with arbitrary value color class

Niichan1403 committed:
Add scroll to the task list

Niichan1403 committed:
Add checkmark button and edit button in Task

Niichan1403 committed:
Add visual effect for checkmark button on click and trash block

Niichan1403 committed:
Add row and column title for eisenhower matrix

Niichan1403 committed:
Add List view

thanksbinh reviewed (approved):
Ok

datleviet commented on package.json:
```diff
@@ -1,8 +1,9 @@
   },
   "dependencies": {
-    "concurrently": "^8.2.0"
+    "concurrently": "^8.2.0",
```
```suggestion
    "concurrently": "^8.2.0"
```

purrion committed:
Remove a global button's style

It's created by create-vite tool and make all buttons have white background in light theme
